aea/connections/stub/connection.py

```diff
@@ -20,10 +20,11 @@
 """This module contains the stub connection."""
 
 import asyncio
+import fcntl
 import logging
 import os
```

Comment:
Note, since you are changing the stub connection code, you will have to bump the stub connection version in its yaml file to 0.2.0. Also all references in the tests and the docs to the stub connection need to be bumped. I can do that once the PR is in a clean state.